packages/wrangler/src/dev-registry.tsx

```diff
@@ -0,0 +1,157 @@
+import http from "http";
+import net from "net";
+import bodyParser from "body-parser";
+import express from "express";
+import { createHttpTerminator } from "http-terminator";
+import { fetch } from "undici";
+import { logger } from "./logger";
+import type { Server } from "http";
+import type { HttpTerminator } from "http-terminator";
+
+const DEV_REGISTRY_PORT = "9999";
```

Comment:
Could probably get away with hopping ports until we find one that's available

Reply:
It has to be a well known port so clients can connect to it (happy to be corrected if I'm wrong here). 999 seems like it may be taken by something tho, so lemme pick something more obscure.

Reply:
changes to 6284

Reply:
Ports can have more than 4 digits...